announcements/1.14.2.md

```diff
@@ -0,0 +1,23 @@
+### Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.14.2.
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+#### New features & improvements
+
+* Android security updates for July 2022 (see [Android Security Bulletin - July 2022](https://source.android.com/security/bulletin/2022-07-01) for more information)
+* Android WebView has been updated to [103.0.5060.71](https://chromereleases.googleblog.com/2022/07/chrome-for-android-update.html)
+* [Join URLs](https://anbox-cloud.github.io/latest/anbox-stream-gateway/#/session/handle-join-session) handed out by the Anbox Stream Gateway to the Anbox container instances will now not expire anymore. This allows sessions to run forever, if needed.
+* The Anbox Cloud Appliance now supports deploying behind a HTTP proxy through the `--proxy` argument available for the `anbox-cloud-appliance init`
```

Comment:
```suggestion
* Included Android security updates for July 2022 (see [Android Security Bulletin - July 2022](https://source.android.com/security/bulletin/2022-07-01) for more information).
* Updated Android WebView to [103.0.5060.71](https://chromereleases.googleblog.com/2022/07/chrome-for-android-update.html).
* [Join URLs](https://anbox-cloud.github.io/latest/anbox-stream-gateway/#/session/handle-join-session) handed out by the Anbox Stream Gateway to the Anbox container instances will now not expire anymore. This allows sessions to run forever, if needed.
* The Anbox Cloud Appliance now supports deploying behind a HTTP proxy through the `--proxy` argument available for the `anbox-cloud-appliance init` command.
```
(Just to be a bit more consistent - periods are needed since the third item has more than one sentence, and then it's better to word the other items so that a period actually makes sense. ;) )